app.js

```diff
@@ -1,7 +1,37 @@
 const express = require("express")
 const { getApi } = require("./controllers/app.controller")
+const { getTopics } = require("./controllers/topics.controller")
 const app = express()
 
+app.use((req, res, next) => 
+    next()
+)
+
 app.get("/api", getApi)
 
+app.get("/api/topics", getTopics)
+
+
+
+app.use('/*', (req, res) => {
+    res.status(404).send({ msg: 'Route not found' })
+})
+
+app.use((err, req, res, next) => {
+    if (err.status){
+      res.status(err.status).send({ msg: err.msg })
```

Comment:
If we're expecting a key of `msg` as well, could we be a little more explicit here? The reason is that every single error object will pass through this in the chain, so it's good to be quite explicit in these if statements!